client/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -21,8 +21,8 @@ describe('CodeEvalResultObjectFactory', function() {
   var codeEvalResult;
   var CODE = "code";
   var OUTPUT = "output";
-  var CORRECTNESS_TEST_RESULTS = "correctnessTestResults";
-  var LAST_TASK_RESULTS = "s";
+  var CORRECTNESS_TEST_RESULTS = ["xyz", "xyz", "xyz", "xyz", "xyz"];
+  var LAST_TASK_RESULTS = "xyz";
   var BUGGY_OUTPUT_TEST_RESULTS = "buggyOutputTestResults";
```

Comment:
I'm not sure I understand what you mean. When you have a moment could you explain it more to me?